src/pages/CatShow.js

```diff
@@ -9,31 +16,62 @@ const CatShow = ({ cats, deleteCat }) => {
 
   return (
     <>
-      <Card className="my-2">
-        <CardImg
+      <div className="profilecontainer">
+        <img
+          className="profilepic"
           alt="Card image cap"
           src={currentCat.image}
+        />
+        <span className="bigname"> {currentCat.name}</span>
+      </div>
+      <div className="bio">
+        <p />
+        🐾 {currentCat.age} years old
+        <p />
+        😻 Enjoys {currentCat.enjoys}
+      </div>
+      <p></p>
```

Comment:
Doesn't look like there is anything in this paragraph tag.  Consider removing it before merging.